### PR TITLE
Unpin magma/mantle/fault

### PR DESCRIPTION
**Pull request by leonardt**
(and peak/lassen also covered in https://github.com/StanfordAHA/GarnetFlow/pull/13)

Using this to track status of the magma 2 integration:
- [x] Merge https://github.com/cdonovick/peak/pull/114
- [x] Merge https://github.com/StanfordAHA/lassen/pull/157
- [x] Merge https://github.com/StanfordAHA/canal/pull/21
- [x] Update gemstone for magma 2 https://github.com/StanfordAHA/gemstone/pull/33
- [x] Update garnet for magma 2 https://github.com/StanfordAHA/garnet/pull/398

### CODE REVIEW

**Comment by Kuree** [comment]
@leonardt Any idea on why it failed?

**Comment by leonardt** [comment]
This isn’t ready yet, see the above checklist, this is opened to track when it is ready

**Comment by leonardt** [comment]
buildkite passed!

File: scripts/repo.json [hunk 1]
```diff
@@ -49,25 +47,25 @@
...
         "url": "https://github.com/phanrahan/peak",
-        "branch": "e958ab4996a49ff19d2c2ed5bdfcffc02a69eb36"
+        "branch": "update-rtl-utils"
     },
     {
         "name": "lassen",
         "type": "git",
         "url": "https://github.com/StanfordAHA/lassen",
-        "branch": "b81035a79182fb075e11dac11c45b2ad1d3d218e"
+        "branch": "fp-mask"
```
Comment (rdaly525) [inline scripts/repo.json:68]: Is fp-mask still correct?

**Comment by leonardt** [comment]
We could merge this (with the peak/lassen/garnet/gemstone branches), then merge the above PRs, then submit a new PR that reverts everything to master.

File: scripts/repo.json (hunk 1 above)
Comment (leonardt) [reply scripts/repo.json:68]: Yes: https://github.com/StanfordAHA/lassen/pull/157 (we changed the name of the PR but kept the branch)

**Pull request merged**